lib/gitscrub.js

```diff
@@ -85,15 +85,21 @@ gitscrub.grabAllReadmes = function(logValue) {
 gitscrub.parseReadMe = function(readme, options, callback) {
   var formattedReadMe = {}
   var header = ""
+  var jotNoteArray = []
   var readMeParsed = Lazy(readme).split("\n")
   readMeParsed.each(function(line) {
-    if (line.substring(0, 2) == "# ") {
+    if (line.substring(0, 2) === "# ") {
       formattedReadMe['title'] = line.substring(2)
-    } else if (line.substring(0, 3) == "## ") {
+    } else if (line.substring(0, 3) === "## ") {
       header = line.substring(3)
-      formattedReadMe[header] = ""
+      jotNoteArray = []
+      formattedReadMe[header] = {text: "", jotNotes: []}
     } else if (header != "") {
-      formattedReadMe[header] += line
+      if (line.substring(0,2) === "* "){
```

Comment:
Good point. One solution might be to strip any preceding whitespace in the line. This would also allow us to potentially categorize jot notes by the indentation level.